### DISCUSSION TITLE
Template for svm_labels

### DISCUSSION THREAD

**Discussion by luddite516** [NetApp/harvest#838] [Show and tell]
I have been using Harvest for a few months now and have built a bunch of dashboards.  I like the fact that Harvest out-of-the-box provides " *_labels" metrics for capturing configuration info for various objects.

volume_labels, shelf_labels, node_labels, etc.

I noticed that Harvest does not provide 'svm_labels', so I developed a template which would do that:

name:               Vserver
query:              vserver-get-iter
object:             svm

counters:
  vserver-info:
    - volume-delete-retention-hours => retention
    - ^comment           => comment
    - ^ipspace           => ipspace
    - ^operational-state => operational_state
    - ^state             => state
    - ^uuid              => uuid
    - ^^vserver-name     => svm
    - ^vserver-type      => type

export_options:
  instance_keys:
    - svm
    - uuid
  instance_labels:
    - comment
    - ipspace
    - operational_state
    - state
    - type

This comes in handy for building SVM lists in Grafana, where I want to show only data SVMs but not other types.

**Comment by cgrinds** [comment]
nice! thanks for sharing @luddite516 that's a nice addition
